spec/controllers/api/v1/project_roles_controller_spec.rb

```diff
@@ -71,6 +71,29 @@
     end
 
     it_behaves_like "is updatable"
+
+    context "mailers" do
+      before(:each) do
+        default_request scopes: scopes, user_id: authorized_user.id
+      end
+
+      # This is the one that should pass, but doesn't
+      it "calls the mailer if user added to project" do
+        params = update_params.merge(id: resource.id)
+        put :update, params
+        #expect(UserAddedToProjectMailerWorker).to receive(:perform_async)
```

Comment:
Thanks, Hound. Would someone mind taking a look at this spec tomorrow? I think I've got the right idea implementation-wise but I'm not sure what the best way to test is. TODO: add similar specs for user controller.